pkg/operator/controllers/gameserverbuild_controller.go

```diff
@@ -207,13 +210,16 @@ func (r *GameServerBuildReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 	// we attempt to create the missing number of game servers, but we don't want to create more than the max
 	// an error channel for the go routines to write errors
 	errCh := make(chan error, maxNumberOfGameServersToAdd)
+
+	// Time how long it takes to trigger new standby gameservers
+	standByReconcileStartTime := time.Now()
```

Comment:
is the purpose of the metric to count how much time it takes to .Create a GameServer? If so, I don't think we need this variable. We can simply get the time.Now() inside the goroutine (line 254) and then emit the metric with the diff. Apologies if I misunderstood the purpose of the metric

Reply:
The name needs to be changed given the other metrics have, but this is supposed to be the time it takes the control plane to create a GameServer object, not the time until it reaches initialization or standingby. So theoretically, this would show issues happening within the controllers themselves, independent of issues that may be related to the type of game someone is trying to spin up

Reply:
All that said, I'm not heavily attached to this metric; if you believe the metrics for reaching initialization and standingby would also be able to show this, then I wouldn't have an issue with removing it

Reply:
Thanks for the clarification. The way this is implemented, it will be emitted once per GameServer creation and for the Nth GameServer, it will also include the time it took to create GameServer N-1 and GameServer N-2 (where N>=0). Is this what we want?
For issues happening with controller themselves, we already emit a metric of reconciliation errors, so I guess we're good on that front (till we aren't :D). 
So, I suggest we remove it till we absolutely need it. What do you think? Appreciate it!

Reply:
I've removed it, but yes. In Grafana, only the last metric would be used (overall batch time). But this way also allows an individual to see the creation times of each GameServer if they wanted to query the metrics in Prometheus; there's a good difference between all GameServers taking 10 seconds to launch versus a single GameServer causing the reconciliation loop to take 10 seconds. But again, I wasn't attached to this and have removed it.